src/components/Login/Login.js

```diff
@@ -32,44 +15,40 @@ const Login = ({ onClose, isOpen, switchPopups }) => {
       title='Sign in'
       name='login'
       isOpen={isOpen}
-      switchPopups={switchPopups}>
+      isValid={isValid}
+      switchPopups={switchPopups}
+      errors={errors}>
       <label className='popup__input-label'>Email</label>
-
       <input
         className='popup__input popup__input_type_login'
         placeholder='Enter email'
         name='email'
-        value={'' || input.email}
-        maxLength='30'
-        type='text'
+        type='email'
         onChange={handleChange}
         required
       />
-
       <span
         id='validation-error'
-        className='popup__error popup__error_visible popup__error_type_login'>
-        Invalid email address
+        className={`popup__error ${
+          errors.email ? `popup__error_visible` : ''
+        }`}>
+        {errors.email}
       </span>
-
       <label className='popup__input-label'>Password</label>
-
       <input
         className='popup__input popup__input_type_login'
         placeholder='Enter password'
         name='password'
-        value={'' || input.password}
         maxLength='30'
         minLength='8'
-        type='text'
+        type='password'
         onChange={handleChange}

```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components